src/components/ReadTime/ReadTime.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import cx from 'classnames';
+import FormattedDate from 'FormattedDate';
+
+type ReadTimeProps = {
+  className?: string;
+  date?: string;
+  metaLabel?: string;
+  readTime?: string;
+};
+
+export const ReadTime = ({
+  className,
+  date,
+  metaLabel,
+  readTime
+}: ReadTimeProps) => {
+  return (
+    <div className={cx('cc-read-time', { [className]: className })}>
```

Comment:
It's perhaps an edge case that readTime is falsy but it might be worth adding logic for handling that in the render.